integrated_channels/moodle/client.py

```diff
@@ -398,8 +428,9 @@ def delete_content_metadata(self, serialized_data):
             return rsp
         moodle_course_id = parsed_response['courses'][0]['id']
         params = {
-            'wsfunction': 'core_course_delete_courses',
-            'courseids[]': moodle_course_id
+            'wsfunction': 'core_course_update_courses',
+            'courses[0][id]': moodle_course_id,
```

Comment:
Not entirely familiar with Moodle functionality but shouldn't this be `courseids[]`?

Reply:
It's `courseids[]` when `wsfunction` is `core_course_delete_courses` where we simply pass an array of ids to be deleted. But in case of `core_course_update_courses` it needs an courses array of objects.